src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRenderer.cs

```diff
@@ -692,7 +691,7 @@ bool ShouldPop()
 
 		void UpdateNavigationBarHidden()
 		{
-			SetNavigationBarHidden(!Shell.GetNavBarIsVisible(_displayedPage), true);
```

Comment:
Are we sure this doesn't break anything? What's the advantage of removing this line?